test/test.sh

```
@@ -1,8 +1,25 @@
+OK=$GREEN"OK"$RESET
+NG=$RED"NG"$RESET
+
+# get script directry 
+SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
+
+# make tmp directry
+TMP_DIR="$SCRIPT_DIR/tmp"
+mkdir -p "$TMP_DIR"
+
+# set timeout commnad
+TIMEOUT=$(command -v gtimeout || command -v timeout)
```

Shellcheck (0.11.0)

[warning] TIMEOUT appears unused. Verify use (or export if used externally).

(SC2034)